crates/rpc/ipc/src/server/mod.rs

```diff
@@ -421,6 +466,28 @@ impl<B, L> Builder<B, L> {
         self
     }
 
+    /// The server enforces backpressure which means that
+    /// `n` messages can be buffered and if the client
+    /// can't keep with up the server.
+    ///
+    /// This `capacity` is applied per connection and
+    /// applies globally on the connection which implies
+    /// all JSON-RPC messages.
+    ///
+    /// For example if a subscription produces plenty of new items
+    /// and the client can't keep up then no new messages are handled.
+    ///
+    /// If this limit is exceeded then the server will "back-off"
+    /// and only accept new messages once the client reads pending messages.
+    ///
+    /// # Panics
+    ///
+    /// Panics if the buffer capacity is 0.
```

Comment:
This is a bit misleading since this function doesn't panic, but I guess we panic somewhere else?

Reply:
yeah, this eventually panics when the channel is created, 
I took this from the ws settings 1:1